Components/Metrics/PointToSurfaceDistance/elxPointToSurfaceDistanceMetric.h

```diff
@@ -0,0 +1,169 @@
+/*=========================================================================
+ *
+ *  Copyright UMC Utrecht and contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef __elxPointToSurfaceDistanceMetric_H__
+#define __elxPointToSurfaceDistanceMetric_H__
```

Comment:
(Minor nitpick) In elastix, as well as in ITK, we no longer have those `__` underscores at the begin and the end of include guard macro names. See https://github.com/SuperElastix/elastix/pull/339 Of course, we can remove them from your header files later, no problem.